modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,232 @@
+
+[id="egress-traffic-control-towards-cluster-nodes_{context}"]
+== Using nodes peer to control egress traffic to cluster nodes
+
+Using the `nodes` peer administrators can control egress traffic from pods to nodes in the cluster. A benefit of this is that you do not have to change the policy when new nodes are added to the cluster.
+
+The following example allows egress traffic to the Kubernetes API server on port `6443` by any of the namespaces with a `restricted`, `confidential`, or `internal` level of security and in the `dept: engr` tenancy.
+
+.Example of ANP `Allow` egress using `nodes` peer
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: egress-security-allow
+spec:
+  egress:
+  - action: Deny
+    to:
+    - nodes:
+        matchExpressions:
+        - key: node-role.kubernetes.io/worker
+          operator: Exists
+  - action: Allow
+    name: allow-to-kubernetes-api-server
+    ports:
+    - portNumber:
+        port: 6443
+        protocol: TCP
+
+    to:
+    - nodes:
+        matchExpressions:
+        - key: node-role.kubernetes.io/control-plane <1>
+          operator: Exists
+    - pods:
+        namespaceSelector:
+          matchLabels:
+            dept: engr <2>
+        podSelector: {}
+  priority: 55
+  subject: <3>
+    namespaces:
+      matchExpressions:
+      - key: security
+        operator: In
+        values: <4>
+        - restricted
+        - confidential
+        - internal
+----
+<1> Specifies a node or set of nodes in the cluster using the `matchExpressions` field. It will select all nodes if the label is present, but left empty.
+<2> Specifies all the pods labeled with `dept: engr`.
+<3> Specifies the subject of the ANP which includes any namespaces that match the labels used by the network policy. The example matches any of the namespaces with `restricted`, `confidential`, or `internal` level of `security`.
+<4> Specifies key/value pairs for `matchExpressions` field.
+====
+
+[id="egress-traffic-control-networks-peer-external-destinations_{context}"]
+== Using networks peer to control egress traffic towards external destinations
+
+Cluster administrators can use CIDR ranges for `networks` peer and apply an egress traffic policy to those nodes.
+
+The following example uses `networks` peer and combines ANP and BANP policies to isolate egress traffic to a range of specified CIDR ranges.
+
+.Example of ANP and BANP using `networks` peers
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: network-as-egress-peer
+spec:
+  priority: 70
+  subject:
+    namespaces: {}
+  egress:
+  - name: "deny-egress-to-external-dns-servers"
+    action: "Deny"
+    to:
+    - networks:
+      - 8.8.8.8/32
+      - 8.8.4.4/32
+      - 208.67.222.222/32
+    - pods:
```

Comment:
this means you have denied egress traffic to also all your pods in your cluster right?, the rule name is not saying that... please help me understand why do we have this part